src/test/java/org/carlspring/cloud/storage/s3fs/util/EnvironmentBuilder.java

```diff
@@ -32,17 +36,19 @@
     {
         Map<String, Object> env;
 
-        String accessKey = System.getenv(ACCESS_KEY);
-        String secretKey = System.getenv(SECRET_KEY);
-        String region = System.getenv(REGION);
-        String protocol = System.getenv(PROTOCOL);
+        String accessKey = getEnvValue(ACCESS_KEY);
+        String secretKey = getEnvValue(SECRET_KEY);
+        String region = getEnvValue(REGION);
+        String protocol = getEnvValue(PROTOCOL);
+        String bucket = getEnvValue(BUCKET_NAME_KEY);
 
-        if (accessKey != null && secretKey != null && region != null && protocol != null)
+        if (accessKey != null && secretKey != null && region != null && protocol != null && bucket != null)
         {
             env = ImmutableMap.<String, Object>builder().put(ACCESS_KEY, accessKey)
                                            .put(SECRET_KEY, secretKey)
                                            .put(REGION, region)
                                            .put(PROTOCOL, protocol)
+                                           .put(BUCKET_NAME_KEY, bucket)
                                            .build();
```

Comment:
If you could just re-align these chained calls, that'll be great! :)

Reply:
fixed